modules/seedtagBidAdapter.js

```diff
@@ -18,8 +18,11 @@ function mapMediaType(seedtagMediaType) {
   else return seedtagMediaType;
 }
 
-function getMediaTypeFromBid(bid) {
-  return bid.mediaTypes && Object.keys(bid.mediaTypes)[0]
+function hasVideoMediaType(bid) {
+  return Object.keys(bid.mediaTypes)
+    .some(function(mt) {
+      return mt === VIDEO;
+    });
```

Comment:
Isn't this equivalent to:

```
bid.mediaTypes && bid.mediaTypes.video
```

Reply:
thank you so much, it’s already committed